src/Http/Livewire/LivewireDatatable.php

```diff
@@ -66,6 +67,7 @@ class LivewireDatatable extends Component
     public $persistSort = true;
     public $persistPerPage = true;
     public $persistFilters = true;
+    public $multisortable = false;
```

Comment:
```suggestion
    public $multisort = false;
```
I think this name makes more sense. `-able` makes more sense when naming interfaces and traits.